### PR TITLE
[Guided Experience] Add Translation For Setup Object Short Title

### PR DESCRIPTION
**Pull request by f4n0**
There were already two methods for translating the Title and Description, but not for Short Title!

### CODE REVIEW

**Review by haoranpb — DISMISSED** [review]
LGTM!

**Comment by f4n0** [comment]
@haoranpb there is a check that failed, but it looks strange to me. do you have any idea?

**Comment by haoranpb** [comment]
Hmm, not sure here. Just checked, it has passed our internal build, so no worries.

I'm still waiting for reviews from my colleagues, it will be merged afterwards 😄

File: Modules/System/Guided Experience/src/Guided Experience/GuidedExperience.Codeunit.al
Comment (haoranpb) [inline Modules/System/Guided Experience/src/Guided Experience/GuidedExperience.Codeunit.al:306]:
```suggestion
    /// <param name="GuidedExperienceType">The type of setup object.</param>
```

Comment (haoranpb) [reply Modules/System/Guided Experience/src/Guided Experience/GuidedExperience.Codeunit.al:306]: a good catch from @darjoo

Comment (f4n0) [reply Modules/System/Guided Experience/src/Guided Experience/GuidedExperience.Codeunit.al:306]: oh right! 
I've copied it from the other procedure, and I've seen that there are other mistakes like this one

**Comment by f4n0** [comment]
> @microsoft-github-policy-service agree

@microsoft-github-policy-service agree

File: Modules/System/Guided Experience/src/Guided Experience/GuidedExperience.Codeunit.al
Comment (haoranpb) [reply Modules/System/Guided Experience/src/Guided Experience/GuidedExperience.Codeunit.al]: haha that's true. didn't even notice them lol

**Comment by f4n0** [comment]
> 

any updates?

**Comment by haoranpb** [comment]
Merged in master last week, porting to 21.x at the moment.

BTW, the translation will be cut for 2048 chars. But that should be enough for the Short title

**Review by haoranpb — APPROVED** [review]

**Comment by JesperSchulz** [comment]
Availability update: We will publish a fix for this issue in the next update for release 21.

Build ID to track: 52713.

**Comment by JesperSchulz** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. Please do not reply to this, as we do not monitor closed issues. If you have follow-up questions or requests, please create a new issue where you reference this one.

Build ID: 52600.

**Pull request merged**
